drivers/ethernet/eth_w5500_socket.c

```diff
@@ -0,0 +1,1057 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Copyright (c) 2024 The Zephyr Project Contributors
```

Comment:
use real email / company name here

Reply:
Isn't this allowed to assign to zephyr contributors, cc: @kartben

Reply:
wasn't aware. thanks for pointing